### PR TITLE
Docs for dapr int/upgrade in k8s mode using GHCR and private registry

### PR DESCRIPTION
**Pull request by pravinpushkar**
Signed-off-by: Pravin Pushkar <ppushkar@microsoft.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Updated docs for `dapr init -k` for GHCR and private registry
Updated docs for `dapr upgrade -k` for private registry

## Issue reference

Please reference the issue this PR will close: #2496 
CLI PR - https://github.com/dapr/cli/pull/984

### CODE REVIEW

File: daprdocs/content/en/reference/cli/dapr-init.md [hunk 1]
```diff
@@ -119,4 +119,15 @@ Use the `--set` flag to configure a set of [Helm Chart values](https://github.co
 
 ```bash
 dapr init -k --set global.tag=1.0.0 --set dapr_operator.logLevel=error
+```
+
+You can also specify a private registry to pull container images from. As of now `dapr init -k` does not use specific images for sentry, operator, placement and sidecar. It relies on only Dapr runtime container image `dapr` for all these images.
```
Comment (msfussell) [inline daprdocs/content/en/reference/cli/dapr-init.md:124]:
```suggestion
You can specify a private registry to pull container images from. 

Note: `dapr init -k` does not use the specific images for Sentry, Operator, Placement and sidecar. It relies on the Dapr runtime container image `dapr` for all these images.
```

Comment (msfussell) [reply daprdocs/content/en/reference/cli/dapr-init.md:124]: I presume the "note" above is a bug that will be fixed and then this comment will be removed, yes?

Comment (pravinpushkar) [reply daprdocs/content/en/reference/cli/dapr-init.md:124]: @msfussell  Thats true. There is already an issue created for this - https://github.com/dapr/dapr/issues/4712

Also, Could you please update label as "waiting for code pr to be merged", as the code PR is under review at the moment.

**Comment by pravinpushkar** [comment]
@msfussell  This PR can be merged as code is merged now.

File: daprdocs/content/en/reference/cli/dapr-init.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ dapr init [flags]
 | `--slim`, `-s`        |                      | `false`       | Exclude placement service, Redis and Zipkin containers from self-hosted installation |
 | `--timeout`           |                      | `300`         | The wait timeout for the Kubernetes installation                                     |
 | `--wait`              |                      | `false`       | Wait for Kubernetes initialization to complete                                       |
-|        N/A              |DAPR_DEFAULT_IMAGE_REGISTRY|          | In self hosted mode, it is used to specify the default container registry to pull images from. When its value is set to `GHCR` or `ghcr` it pulls the required images from Github container registry. To default to Docker hub as default, just unset this env variable.|
+|        N/A              |DAPR_DEFAULT_IMAGE_REGISTRY|          | It is used to specify the default container registry to pull images from. When its value is set to `GHCR` or `ghcr` it pulls the required images from Github container registry. To default to Docker hub as default, just unset this env variable.|
```
Comment (greenie-msft) [inline daprdocs/content/en/reference/cli/dapr-init.md]:
```suggestion
|        N/A              |DAPR_DEFAULT_IMAGE_REGISTRY|          | It is used to specify the default container registry to pull images from. When its value is set to `GHCR` or `ghcr` it pulls the required images from Github container registry. To default to Docker hub, unset the environment variable or leave it blank|
```

**Comment by greenie-msft** [comment]
@pravinpushkar please see minor suggestion. Thanks

**Comment by pravinpushkar** [comment]
> @pravinpushkar please see minor suggestion. Thanks

Fixed.

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**